frontend/src/components/__tests__/ExternalResourceModal.js

```diff
@@ -132,4 +133,28 @@ describe('utility functions', () => {
     // Then we see it is not
     expect(isExternalURL(url)).not.toBeTruthy();
   });
+
+  it('utility function correctly validates internal urls', () => {
+    process.env.TTA_SMART_HUB_URI = 'https://shrek.com';
+    // Given an internal url
+    const internal = join(process.env.TTA_SMART_HUB_URI, 'some-internal', 'url');
+
+    // When we check if its valid
+    // Then we see it is
+    expect(isValidURL(internal)).toBeTruthy();
+  });
+
+  it('utility function correctly validates other govemernt urls', () => {
+    process.env.TTA_SMART_HUB_URI = 'https://shrek.com';
+
+    const urls = ['shrek', 'fiona', 'donkey'];
+
+    // Given an internal url
+    urls.forEach((url) => {
+      const internal = join(`${url}${GOVERMENT_HOSTNAME_EXTENSION}`, 'some-internal', 'url');
+      // When we check if its valid
+      // Then we see it is
+      expect(isExternalURL(internal)).toBeTruthy();
```

Comment:
These urls should return `isValid` true, but `isExternalURL` false.

Reply:
Ah dang, I ended up solving the issue a different way. I can refactor to make sure those cases pass